src/core_plugins/kibana/common/tutorials/heartbeat_instructions.js

```diff
@@ -0,0 +1,661 @@
+    values: { configureLink: '{config.docs.beats.heartbeat}/heartbeat-configuration.html' }
+  });
+  return {
+    OSX: {
+      title: defaultTitle,
+      textPre: i18n.translate('kbn.common.tutorials.heartbeatEnableCloudInstructions.osxTextPre', {
+        defaultMessage: 'Modify the settings in the `heartbeat.yml` file.',
+      }),
+      commands: defaultCommands,
+      textPost: defaultTextPost
+    },
+    DEB: {
+      title: defaultTitle,
+      textPre: i18n.translate('kbn.common.tutorials.heartbeatEnableCloudInstructions.debTextPre', {
+        defaultMessage: 'Modify the monitors in the `/etc/heartbeat/heartbeat.yml` file.',
+      }),
+      commands: defaultCommands,
+      textPost: defaultTextPost
+    },
+    RPM: {
+      title: defaultTitle,
+      textPre: i18n.translate('kbn.common.tutorials.heartbeatEnableCloudInstructions.rpmTextPre', {
+        defaultMessage: 'Modify the monitors in the `/etc/heartbeat/heartbeat.yml` file.',
+      }),
+      commands: defaultCommands,
+      textPost: defaultTextPost
+    },
+    WINDOWS: {
+      title: defaultTitle,
+      textPre: i18n.translate('kbn.common.tutorials.heartbeatEnableCloudInstructions.windowsTextPre', {
+        defaultMessage: 'From the {path} folder, run:',
+        values: { path: `C:\\Program Files\\Heartbeat` },
+      }),
+      commands: defaultCommands,
+      textPost: defaultTextPost
+    }
+  };
+}
+
+export function heartbeatStatusCheck() {
+  return {
+    title: i18n.translate('kbn.common.tutorials.heartbeatStatusCheck.title', {
+      defaultMessage: 'Heartbeat status',
+    }),
+    text: i18n.translate('kbn.common.tutorials.heartbeatStatusCheck.text', {
+      defaultMessage: 'Check that data is received from the Heartbeat',
+    }),
+    btnLabel: i18n.translate('kbn.common.tutorials.heartbeatStatusCheck.buttonLabel', {
+      defaultMessage: 'Check data',
+    }),
+    success: i18n.translate('kbn.common.tutorials.heartbeatStatusCheck.successText', {
+      defaultMessage: 'Data successfully received from Heartbeat',
+    }),
+    error: i18n.translate('kbn.common.tutorials.heartbeatStatusCheck.errorText', {
+      defaultMessage: 'No data has been received from Heartbeat yet',
+    }),
+    esHitsCheck: {
+      index: 'heartbeat-*',
+      query: {
+        match_all: {},
+      },
+    },
+  };
+}
+
+export function onPremInstructions(platforms, geoipRequired, uaRequired, context) {
+  const HEARTBEAT_INSTRUCTIONS = createHeartbeatInstructions(context);
+
+  return {
+    instructionSets: [
+      {
+        title: i18n.translate('kbn.common.tutorials.heartbeat.premInstructions.gettingStarted.title', {
+          defaultMessage: 'Getting Started',
+        }),
+        instructionVariants: [
+          {
+            id: INSTRUCTION_VARIANT.OSX,
+            instructions: [
+              HEARTBEAT_INSTRUCTIONS.INSTALL.OSX,
+              HEARTBEAT_INSTRUCTIONS.CONFIG.OSX,
+	      heartbeatEnableInstructionsOnPrem().OSX,
```

Comment:
Indentation slightly off here and in the next instances?

Reply:
fixed